cardano-node/src/Cardano/Node/Tracing/Tracers/ChainDB.hs

```diff
@@ -680,7 +745,14 @@ instance MetaTrace  (ChainDB.TraceAddBlockEvent blk) where
         , ( "epoch"
           , "In which epoch is the tip of the current chain."
           )
+        , ( "forks"
+          , "counter for forks"
+          )
+        , ( "tipBlock"
+          , "Hash vallues for tipBlockHash, tipBlockParentHash and tipBlockIssuerVkHashText"
```

Comment:
* typo `values`
* `tipBlockIssuerVkHashText` is not the field actual field name, it's the name you chose for the let binding. Should be `tipBlockIssuerVKeyHash` to match the trace msg.

Reply:
Fixed